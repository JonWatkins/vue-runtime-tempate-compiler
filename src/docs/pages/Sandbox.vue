<template>
  <div class="page">
    <h1 id="top" class="display-4 mb-3">
      <router-link to="/sandbox#top" class="header-anchor">
        <span class="header-anchor--hash">#</span>
        Sandbox
      </router-link>
    </h1>

    <div class="alert alert-primary mb-4">Coming soon...</div>

    <PageNav>
      <template v-slot:left>
        <router-link to="/api-reference">
          <i class="fas fa-arrow-left"></i>
          API Reference
        </router-link>
      </template>
      <template v-slot:right>
        <router-link to="/">
          Introduction
          <i class="fas fa-arrow-right"></i>
        </router-link>
      </template>
    </PageNav>
  </div>
</template>

<script>
import PageNav from '../components/PageNav.vue'

export default {
  name: 'Sandbox',

  components: {
    PageNav
  }
}
</script>
