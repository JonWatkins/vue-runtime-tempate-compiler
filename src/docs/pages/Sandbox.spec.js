import { shallowMount } from '@vue/test-utils'
import Sandbox from './Sandbox.vue'
const { localVueInst } = global

let wrapper

describe('Sandbox', () => {
  beforeEach(() => {
    wrapper = shallowMount(Sandbox, {
      localVue: localVueInst
    })
  })

  it('is called', () => {
    expect(wrapper.name()).toBe('Sandbox')
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
