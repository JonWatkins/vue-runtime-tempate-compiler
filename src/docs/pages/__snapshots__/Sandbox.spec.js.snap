// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sandbox render correctly 1`] = `
<div class="page">
  <h1 id="top" class="mb-3">
    <router-link-stub to="/sandbox#top" tag="a" event="click" class="header-anchor"><span class="header-anchor--hash">#</span>
      Sandbox
    </router-link-stub>
  </h1>
  <div class="alert alert-primary mb-4">Coming soon...</div>
  <pagenav-stub></pagenav-stub>
</div>
`;
